template.rb

```diff
@@ -57,11 +57,7 @@ def apply_template!
     binstubs = %w[brakeman bundler bundler-audit rubocop sidekiq]
     run_with_clean_bundler_env "bundle binstubs #{binstubs.join(' ')} --force"
 
-    if File.exist?("bin/dev")
-      append_to_file "Procfile.dev", "worker: bin/sidekiq"
```

Comment:
No need to run Sidekiq in development (ActiveJob is configured to only use it in production).